E508%20Framework.VR/init.sqf

```diff
@@ -24,9 +24,4 @@
 // ====================================================================================
 		
 //BRIEFING//
-	[] execVM "briefing.sqf";
-
-// ====================================================================================
-	
-//ADDON SETTINGS//
-	[] execVM "addonsettings.sqf";
```

Comment:
Not needed if file is called `cba_settings.sqf`.

https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#mission-settings